### PR TITLE
Adding pytray package

### PR DESCRIPTION
**Pull request by muhrin**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@conda-forge/help-python
@conda-forge-admin, please ping team

I'm willing to be listed as maintainer.

### CODE REVIEW

File: recipes/pytray/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - async_generator
+    - deprecation
+    - python >=3.5
+
+test:
+  imports:
+    - pytray
+
+about:
+  home: https://github.com/muhrin/pytray.git
```
Comment (synapticarbors) [inline recipes/pytray/meta.yaml:32]: This url just redirects to `https://github.com/muhrin/pytray` so let's change it.

File: recipes/pytray/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  summary: A python tools library for baking pies
```
Comment (synapticarbors) [inline recipes/pytray/meta.yaml:33]: Can we have a description that actually says what this package does?

File: recipes/pytray/meta.yaml (hunk 1 above)
Comment (muhrin) [reply recipes/pytray/meta.yaml]: Done.

File: recipes/pytray/meta.yaml (hunk 2 above)
Comment (muhrin) [reply recipes/pytray/meta.yaml:33]: Done.

**Pull request merged**